cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -279,14 +279,14 @@ public ServiceTicket grantServiceTicket(
         getAuthenticationSatisfiedByPolicy(ticketGrantingTicket, new ServiceContext(service, registeredService));
 
         final String uniqueTicketIdGenKey = service.getClass().getName();
-        if (!this.uniqueTicketIdGeneratorsForService.containsKey(uniqueTicketIdGenKey)) {
-            logger.warn("Cannot create service ticket because the key [{}] for service [{}] is not linked to a ticket id generator",
-                    uniqueTicketIdGenKey, service.getId());
-            throw new UnauthorizedSsoServiceException();
-        }
-        
-        final UniqueTicketIdGenerator serviceTicketUniqueTicketIdGenerator =
+        logger.debug("Looking up service ticket id generator for [{}]", uniqueTicketIdGenKey);

```

Comment:
I updated the issue in the description to clarify the use case here. This behavior is useful and intended to happen in order to not require explicit generators, specially in cases where they are not cared for. Suppose I created a new service and then an argument extractor. I dont want to, or should not have to create a generator for my service if I have no special requirements for it. That would just be boilerplate configuration noise. While my service could do something processing on the request, my generator would not necessarily have to be all that special to require something explicitly being there.